pkg/extension/sumologicextension/README.md

```diff
@@ -28,9 +28,7 @@ and can be used as an authenticator for the
 
 ## Configuration
 
-- `access_id`: (required) access ID for Sumo Logic service, see
-  [help][credentials_help] for more details
-- `access_key`: (required) access key for Sumo Logic service, see
```

Comment:
Can we perhaps make it crossed out for now and marked as deleted? Perhaps that would help users that use it as of now 🤔